app/assets/javascripts/libfigure/param.js

```diff
@@ -1,13 +1,13 @@
-//     ____   _    ____      _    __  __ 
+//     ____   _    ____      _    __  __
 //    |  _ \ / \  |  _ \    / \  |  \/  |
 //    | |_) / _ \ | |_) |  / _ \ | |\/| |
 //    |  __/ ___ \|  _ <  / ___ \| |  | |
 //    |_| /_/   \_\_| \_\/_/   \_\_|  |_|
 //
 // Params have semantic value specific to each figure.
 // Though some patterns have emerged. Patterns like:
-// figures have a subject telling who's acted on by the figure. 
-// 
+// figures have a subject telling who's acted on by the figure.
+//
```

Comment:
Ah, Atom's penchant for removing trailing whitespace and generating lots of diffs. I wish I knew how to make emacs do the same, so that eventually all files were trailing-whitespace-free.